cmd/koord-manager/main.go

```diff
@@ -36,8 +36,11 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 
```

Comment:
I think we need to keep `_ "k8s.io/client-go/plugin/pkg/client/auth"`. Since by default, plugins that use [client-go](https://godoc.org/k8s.io/client-go) cannot authenticate to Kubernetes clusters on many cloud providers.